### PR TITLE
Tweak Vector math page (style, links, more notes)

### PR DESCRIPTION
**Pull request by akien-mga**
Also cleanup style issues in the docs image guidelines.

### CODE REVIEW

File: contributing/documentation/docs_image_guidelines.rst [hunk 1]
```diff
@@ -129,8 +129,15 @@ multiple outlines in multiple colors.
...
+Once you've finished working on your image, it can be added to the documentation.
+All images are stored in folders named ``img`` next to the page they are used in.
+
+To add your image, add it to the ``img`` folder that's in the same folder as the
+``.rst`` file for the page (create it if it doesn't exist). In the ``.rst`` page,
+images should be included with the following code snippet::
+
+    .. image:: img/documentation_image.webp
+
+Where ``documentation_image.webp`` would be change to the name of the image you
```
Comment (Piralein) [inline contributing/documentation/docs_image_guidelines.rst:141]:
```suggestion
Where ``documentation_image.webp`` would be changed to the name of the image you
```

**Review by Piralein — CHANGES_REQUESTED** [review]

File: tutorials/math/vector_math.rst [hunk 2]
```diff
@@ -220,16 +230,16 @@ other object:
...
+Here is a code example of the above diagram using a :ref:`CharacterBody2D
 <class_CharacterBody2D>`:
 
-
 .. tabs::
  .. code-tab:: gdscript GDScript
 
-    # object "collision" contains information about the collision
+    # The KinematicCollision2D object returned by `move_and_collide` contains
+    # information about the collision.
```
Comment (mhilbrunner) [inline tutorials/math/vector_math.rst:242]: I wonder if its worth it to move this comment to the descriptive text, both to be able to link KinematicCollision2D and for the sake of translations. Also I agree this is an improvement over the original, but it starts to get a bit wordy for a comment compared to the length of the code.

Same for the C# version obviously. :)

File: contributing/documentation/docs_image_guidelines.rst (hunk 1 above)
Comment (akien-mga) [reply contributing/documentation/docs_image_guidelines.rst]: Fixed.

**Review by Piralein — APPROVED** [review]
